styles.css

```diff
@@ -10,6 +10,7 @@
 	html,
 	body {
 		@apply scroll-smooth;
+        font-family: --var(open-sans-font), Helvetica Neue, Helvetica, Arial, sans-serif;
```

Comment:
@wicksipedia is this fine with the next/font additions you made? same with the changes in tailwind.config.js